scripts/reprocessing/buildClowderInstanceLocalTest.py

```diff
@@ -0,0 +1,120 @@
+import os
+import datetime
+import json
+
+from pyclowder.connectors import Connector
+from pyclowder.datasets import upload_metadata
+from pyclowder.files import upload_to_dataset
+from terrautils.extractors import build_dataset_hierarchy, build_metadata
+from terrautils.metadata import clean_metadata
+
+
+# CONNECTION SETTINGS
+CLOWDER_HOST = "http://141.142.22.37:9000/"
+CLOWDER_KEY = "r1ek3rs"
```

Comment:
is this a secret key?

Reply:
it's the default secret key for clowder in the clowder code - you only have this one if you run a vanilla docker container. production instances each set their own.